recipes/sspace/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+mv * $PREFIX/bin
+ln -s $PREFIX/bin/SSPACE_Standard_v3.0.pl $PREFIX/bin/SSPACE
+cd $PREFIX/bin/dotlib/
+wget -c http://cpansearch.perl.org/src/GBARR/perl5.005_03/lib/getopts.pl
```

Comment:
@bioconda/perl any advice? Does `conda skeleton cran` only work with `perl` and not `perl-threaded`?